lib/rust/parser/doc-parser/src/doc_sections.rs

```diff
@@ -41,7 +41,6 @@ impl DocParser {
     }
 
     /// Parse the documentation.
-    #[profile(Detail)]
```

Comment:
This was the ONLY remaining usage of the `enso-profiler`, so I just removed it. We don't need this anymore, since we use the parse function directly anyway, and the whole profiler machinery was effectively abandoned once we removed old gui.